source/main.coffee

```diff
@@ -161,7 +161,7 @@ export compile = (src, options) ->
 
   if options.errors?.length
     #TODO: Better error display
-    throw new Error "Parse errors: #{options.errors.join("\n")} "
+    throw new Error "Parse errors: #{options.errors.map((e) -> e.message).join("\n")} "
```

Comment:
Do these come through on the LSP now too? It'd be especially cool if we could have error recovery in addition to reporting. Like we return an error in the parse tree but the parser keeps going. I'm thinking of `foo.` with no property after, to enable autocompletion.

Reply:
They're not surfaced in the LSP yet but that's the planned direction.